daemon/cluster/executor/container/container.go

```diff
@@ -628,34 +628,35 @@ func (c *containerConfig) serviceConfig() *clustertypes.ServiceConfig {
 func networkCreateRequest(name string, nw *api.Network) clustertypes.NetworkCreateRequest {
 	ipv4Enabled := true
 	ipv6Enabled := nw.Spec.Ipv6Enabled
-	options := network.CreateOptions{
-		// ID:     nw.ID,
-		Labels:     nw.Spec.Annotations.Labels,
+	networkCreateRequest := network.CreateRequest{
```

Comment:
really nit: perhaps a shorter name (`req`) would work here; ultimately, we know we're creating a request here 😅